apps/etterlatte-behandling/src/main/kotlin/oppgaveny/OppgaveServiceNy.kt

```diff
@@ -31,11 +31,26 @@ class OppgaveServiceNy(private val oppgaveDaoNy: OppgaveDaoNy, private val sakDa
         }
     }
 
-    fun byttSaksbehandler(nySaksbehandlerDto: NySaksbehandlerDto) {
+    fun byttSaksbehandler(saksbehandlerEndringDto: SaksbehandlerEndringDto) {
         inTransaction {
-            val hentetOppgave = oppgaveDaoNy.hentOppgave(nySaksbehandlerDto.oppgaveId)
+            val hentetOppgave = oppgaveDaoNy.hentOppgave(saksbehandlerEndringDto.oppgaveId)
             if (hentetOppgave != null) {
-                oppgaveDaoNy.settNySaksbehandler(nySaksbehandlerDto)
+                oppgaveDaoNy.settNySaksbehandler(saksbehandlerEndringDto)
+            } else {
+                throw NotFoundException("Oppgaven finnes ikke")
+            }
+        }
+    }
+
+    fun fjernSaksbehandler(oppgaveId: OppgaveId) {
+        inTransaction {
+            val hentetOppgave = oppgaveDaoNy.hentOppgave(oppgaveId.oppgaveId)
+            if (hentetOppgave != null) {
+                if (hentetOppgave.saksbehandler != null) {
+                    oppgaveDaoNy.fjernSaksbehandler(oppgaveId.oppgaveId)
+                } else {
+                    throw BadRequestException("Oppgaven har ingen saksbehandler")
```

Comment:
Foreslår å også ta med oppgave-id i feilmeldinga for å gjera feilsøking enklare

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1765/commits/da5db45f156eaffa22aee556c207da8f72a9c49f